packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -133,51 +134,36 @@ export interface EfsFileSystemProps {
    *
    * @default - none
    */
-  readonly lifecyclePolicy?: EfsLifecyclePolicyProperty;
+  readonly lifecyclePolicy?: LifecyclePolicyProperty;
```

Comment:
If you're cleaning up and breaking things anyway, can you rename this to `LifecyclePolicy` as well?